content/blog/trustless-lightning-payments-for-content.md

```diff
@@ -0,0 +1,357 @@
+---
+
+title: "Trustless Lightning Payments for Content"
+subtitle: "How PTLC payment channels in Lightning enable decentralized social media."
+date: 2022-01-07T00:00:00+00:00
```

Comment:
We might want to move this a bit closer to the actual publishing date once this is ready for merge.

Reply:
Will do.